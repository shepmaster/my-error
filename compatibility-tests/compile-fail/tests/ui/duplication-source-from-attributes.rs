use snafu::Snafu;

#[derive(Debug, Snafu)]
enum EnumError {
    AVariant {
        // Should detect second attribute as duplicate
        #[snafu(source(from(EnumError, Box::new)))]
        #[snafu(source(from(EnumError, Box::new)))]
        my_source: Box<EnumError>,
    },
}

fn main() {}
