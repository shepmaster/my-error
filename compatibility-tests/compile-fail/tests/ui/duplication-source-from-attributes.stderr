error: Multiple `source` attributes are not supported on a field
 --> $DIR/duplication-source-from-attributes.rs:8:17
  |
8 |         #[snafu(source(from(EnumError, Box::new)))]
  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
